### DISCUSSION TITLE
How to Enable and Configure Ignition Modifier in Valheim Ashlands using Docker

### DISCUSSION THREAD

**Discussion by mvladk** [mbround18/valheim-docker#847] [Q&A] [answered]
Hello everyone,

I'm running a Valheim server using the Docker image provided by mbround18, and I'm trying to enable the "ignition modifier" in the Ashlands update. I'm facing an issue where my config.json file reverts to its default settings every time the Docker container restarts.

Here are a few details about my setup:

Docker Image Used: mbround18/valheim:latest
Configuration Attempts: I've tried modifying the config.json directly within the container, but it doesn't seem to persist after a restart.
I have a few questions:

Is there a recommended way to ensure that changes to config.json persist through container restarts? Should these settings be integrated into the docker-compose.yml file instead?
What is the correct parameter or setting to enable and manage the ignition modifier within the Docker setup?
Any advice or examples of how to manage this configuration would be greatly appreciated!

Thank you!

**Comment by mbround18** [comment]
Can i get a copy of your docker compose? config.json  is written upon docker compose up, for modifiers it should be this syntax

```yaml
    environment:
      - "MODIFIERS=
        raids=muchmore,
        combat=hard,
        deathpenalty=casual"
```

**Reply by mvladk** [reply]
This I've seen on readme, but what os exact parameter for Ignition?

**Reply by mbround18** [reply]
![how-to-set-values](https://github.com/mbround18/valheim-docker/assets/12646562/c9dec695-2826-4b4e-ad7a-69a979e358f8)

Small caveat with SET_KEY, i need to allow it to process  multiple. thats a feature im working on rn

**Reply by mvladk** [reply]
Thanks @mbround18 i tried without success:

- SET_KEY=firehazards

Here is full docker compose
cat docker-compose.yml 


version: "3"
services:
  valheim:
    image: mbround18/valheim:latest
    ports:
      - 2456:2456/udp
      - 2457:2457/udp
      - 2458:2458/udp
    environment:
      - PORT=2456
      - NAME=MyServer
      - WORLD=Dedicated
      - PASSWORD=PASSWORD
      - TZ=Europe/London
      - PUBLIC=1
      - MODIFIERS="combat=hard"
      - SET_KEY=firehazards
      - AUTO_UPDATE=1
      - AUTO_UPDATE_SCHEDULE=0 1 * * *
      - AUTO_BACKUP=1
      - AUTO_BACKUP_SCHEDULE=0 0 * * *
      - AUTO_BACKUP_REMOVE_OLD=1
      - AUTO_BACKUP_DAYS_TO_LIVE=3
      - AUTO_BACKUP_ON_UPDATE=1
      - AUTO_BACKUP_ON_SHUTDOWN=1
      - FIRE_DAMAGE_ENABLED=1
      - FIRE_DAMAGE_AMOUNT=0.5
      - ENVIRONMENTAL_DAMAGE_ENABLED=1
      - FIRE_DAMAGE=true
      - FROST_DAMAGE=true
      - POISON_DAMAGE=true
      - RAID_DIFFICULTY=hard
      - ENEMY_HEALTH_MULTIPLIER=1.5
      - ENEMY_DAMAGE_MULTIPLIER=1.5
      - CROSSPLAY=1
    volumes:
      - ./valheim/saves:/home/steam/.config/unity3d/IronGate/Valheim
      - ./valheim/server:/home/steam/valheim
      - ./valheim/backups:/home/steam/backups

**Reply by mbround18** [reply]
I have asked in the Valheim discord what the flag is awaiting response

**Reply by mbround18** [reply, marked as answer]
@mvladk just got a response, try the value `fire`

**Reply by mvladk** [reply]
Thanks, fire worked:

`- SET_KEY=fire`
